group/views.py

```diff
@@ -95,64 +76,43 @@ def get_all_announce(request):
             })
 
     
-def detail(request,group_id):
-    running_contest_list = [
-        {'name':'11111', 's_time':'20111010', 'e_time':'20111010'},
-        {'name':'22222', 's_time':'20140505', 'e_time':'20111010'},
-        {'name':'33333', 's_time':'20122222', 'e_time':'20111010'},
-        {'name':'44444', 's_time':'20133333', 'e_time':'20111010'},
-        {'name':'55555', 's_time':'20144444', 'e_time':'20111010'},
-    ]
-
-    ended_contest_list = [
-        {'name':'aaaaa', 'time':'20111010'},
-        {'name':'bbbbb', 'time':'20140505'},
-        {'name':'ccccc', 'time':'20122222'},
-        {'name':'ddddd', 'time':'20133333'},
-        {'name':'eeeee', 'time':'20144444'},
-    ]
-
-    annowence_list = [
-        {'id':'1', 'name':'annowence_1', 'op':'drowsy', 'time':'20111010', 'content':'asdasd'},
-        {'id':'2', 'name':'annowence_2', 'op':'drowsy', 'time':'20140505', 'content':'qwefad'},
-    ]
-
-    student_list =[
-        {'name': 'Tom', 'time': '2014/1/2 23:00'},
-        {'name': 'Amy', 'time': '2014/1/2 23:00'},
-        {'name': 'Jess', 'time': '2014/1/2 23:00'},
-        {'name': 'Bieber', 'time': '2014/1/2 23:00'},
-        {'name': 'Ian', 'time': '2014/1/2 23:00'},
-        {'name': 'Anthony', 'time': '2014/1/2 23:00'},
-        {'name': 'Lily', 'time': '2014/1/2 23:00'}
-    ]
-
-    ta_list =[
-        {'name': 'drowsy'},
-        {'name': 'henry'},
-    ]
+def detail(request, group_id):
+    
+    group = Group.objects.get(id = group_id)
+    all_contest = group.trace_contest.all()[0:5]
```

Comment:
I prefer using [order_by](https://docs.djangoproject.com/en/1.7/ref/models/querysets/#django.db.models.query.QuerySet.order_by) instead of all.